src/operations/list_collections.ts

```diff
@@ -90,14 +90,17 @@ export class ListCollectionsOperation extends CommandOperation<string[]> {
       return;
     }
 
-    const command = {
+    return super.executeCommand(server, session, this.generateCommand(), callback);
+  }
+
+  /** @internal */
+  generateCommand(): Document {
```

Comment:
Sounds good, can you put that in the docs string comment, you can drop `@internal` since the whole class is internal, but it can't hurt to say this was written for unit testing. 
(My thinking: I can imagine a future where generateCommand is defined on every operation class and that would just be intentional design as opposed to just help test.)

Reply:
Updated the comment to note the method was added for testing. I agree - I was thinking the same in the future that all the commands could have it to starting easing the ability to unit test them.